unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExploreHUD/Scripts/HighlightScenes/HotSceneCellView.cs

```diff
@@ -117,16 +118,22 @@ public void Clear()
     public void JumpInPressed()
     {
         HotScenesController.HotSceneInfo.Realm realm = new HotScenesController.HotSceneInfo.Realm() { layer = null, serverName = null };
+        hotSceneInfo.realms = hotSceneInfo.realms.ToList().OrderByDescending(x => x.usersCount).ToArray();
+        List<HotScenesController.HotSceneInfo.Realm> nextMostPopulatedRealms = new List<HotScenesController.HotSceneInfo.Realm>();
         for (int i = 0; i < hotSceneInfo.realms.Length; i++)
         {
             if (hotSceneInfo.realms[i].usersCount < hotSceneInfo.realms[i].usersMax)
```

Comment:
Yes, we could directly send the list of ordered realms instead of the main realm + the list of the next ones 👍 I'm going to take a look to it, thanks!

Reply:
This feedback no longer applies because we have refactorized the implementation to bring responsibilities from Kernel to Unity.